src/main/scala/mesosphere/marathon/api/ModelValidation.scala

```diff
@@ -288,3 +288,5 @@ trait ModelValidation extends BeanValidation {
   }
 
 }
+
+object ModelValidation extends ModelValidation
```

Comment:
This is only being used in the test. Can we instead just create an anonymous instance in the test?